### ISSUE TITLE
Rotate monitor (counter)clockwise to portrait

### ISSUE THREAD

**Issue by hendymen** [OpenSurv/opensurv#5]
I've just got to know OpenSurv, really liking the speed of it!

However i've got a monitor connected and mounted counterclockwise in portrait mode.

I don't see an possibility to change the orientation of the monitor in the configuration YML's.

Also i've tried with xandr, but it doesn't seem to see the display?

![image](https://github.com/user-attachments/assets/d848437f-0f5a-4151-ad21-b4cc57d006f6)

**Comment by SvenVD** [comment]
Good idea, I added an option for a screen ( and all streams within ) to be rotated 90 degrees. Please check it out at https://github.com/OpenSurv/opensurv/tree/v1_latest .

**Comment by SvenVD** [comment]
What is the hardware you are currently running on?

**Comment by hendymen** [comment]
This worked like a charm!

![Afbeelding van WhatsApp op 2024-11-12 om 21 00 11_995cf873](https://github.com/user-attachments/assets/75be7009-728d-4cdf-8990-7b52fff00487)

I'm running it on an RPI 4, 8GB ram.

3* 720p H.265 rtsp streams at the moment without a issue!

Thanks again!
